<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>511</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabSettings">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="4">
      <widget class="QSpinBox" name="totalMemorySpinBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Total memory</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Steps execution speed</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QDoubleSpinBox" name="stepsExecSpeedSpinBox"/>
     </item>
     <item row="0" column="4">
      <widget class="QSpinBox" name="minBlockSizeSpinBox"/>
     </item>
     <item row="0" column="3">
      <widget class="QSlider" name="minBlockSizeSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="totalMemoryLabel">
       <property name="text">
        <string>VALUE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QSlider" name="stepsExecSpeedSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="2">
      <widget class="QComboBox" name="drawingTool"/>
     </item>
     <item row="1" column="3">
      <widget class="QSlider" name="totalMemorySlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="minBlockSizeLabel">
       <property name="text">
        <string>VALUE</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Min. block size</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>GraphVix drawing tool</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QCheckBox" name="autoSave">
       <property name="text">
        <string>Autosave commands</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3" colspan="2">
      <widget class="QPushButton" name="restoreDefaultSettings">
       <property name="text">
        <string>Restore default settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tabCommands">
    <attribute name="title">
     <string>Commands</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>183</width>
       <height>299</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Execution</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="autoExec">
           <property name="text">
            <string>Automatic execution</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="execNextCmd">
           <property name="text">
            <string>Execute next</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="resetExec">
           <property name="text">
            <string>Reset execution</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Commands</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QPushButton" name="saveCmds">
           <property name="text">
            <string>Save commands</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="loadCmds">
           <property name="text">
            <string>Load commands</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>331</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabLog">
    <attribute name="title">
     <string>Log</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
